_docs-sources/foundations/landing-zone/manage-accounts.md

```diff
@@ -35,40 +35,32 @@ Below you'll find a table with common AWS account operations and the Gruntwork r
 
 ### Remove an AWS account
 
-We recommend using the AWS Console's Organizations interface for closing an account and destroying all of its resources, due to current flakiness of the AWS Service Catalog when performing deletes via OpenTofu/Terraform providers. See the steps below for the complete sequence of actions to remove an account.
+We recommend the following three step procedure to close an AWS account managed by Devops Foundations:
 
-:::note
-AWS Organizations will permanently [close an account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html) after 90 days.
-:::
+1. [Cleanup the OpenTofu/Terraform State for the Control Tower module](#1-cleanup-opentofuterraform-state-for-the-control-tower-module)
+1. [Cleanup Infrastructure Code](#2-cleanup-infrastructure-code)
+1. [Close the Account with Clickops](#3-close-the-accounts-in-aws-organizations)
 
 
+#### 1. Cleanup the OpenTofu/Terraform State for the Control Tower module
 
-#### 1. (Optional) Destroy resources in the account to be removed
-
-You may choose or need to destroy some or all of your provisioned resources, if the resources are managed by Terragrunt in your infrastructure repository which has Gruntwork Pipelines integrated, you can destroy them by doing the following:
-
-1. Checkout a new branch in the infrastructure repository
-1. Remove the folder(s) containing the resources you want to destroy
-1. Commit your changes and create a Pull Request which will trigger Gruntwork Pipelines to destroy the resource once the Pull Request is merged.
-
-    :::danger
-    Make sure you are deleting the correct resources in the right account and that you have a backup of any data you may need in the future.
-
-    It is not recommended that you delete the AWS IAM OIDC provider or the AWS IAM roles used by Pipelines within the AWS account before closing the account. This ensures that you can continue to use Pipelines to manage resources within it (including destroying resources) prior to its full closure.
-    :::
+1. Obtain AWS CLI credentials for the your *management* account.
+1. Remove the targeted account(s) from being controlled by the *management* account's Control Tower module by running the following commands:
 
-    :::note
-    You may also need to delete resources if you encounter issues with coordination between different accounts where resources remain present in the account to be closed that are referenced in other accounts.
-    :::
+  ```bash
+  cd management/_global/control-tower-multi-account-factory/ # Navigate to the Control Tower module directory in your central/root infrastructure repository
+  terragrunt plan # Observe that targeted accounts are set for destruction
```

Comment:
@oredavids if this is step 1, then the plan would show no changes, right?  Do we need to plan/apply at all here?

Reply:
I realized that splitting the state modification from Iac cleanup is not ideal, this has been shuffled back into IaC cleanup and the comments make sense again.